lib/msal-browser/src/config/Configuration.ts

```diff
@@ -161,10 +167,18 @@ export function buildConfiguration({ auth: userInputAuth, cache: userInputCache,
         allowRedirectInIframe: false
     };
 
+    const DEFAULT_TELEMETRY_OPTIONS: Required<BrowserTelemetryOptions> = {
+        clientTelemetry: {
```

Comment:
Instead of `telemetry.clientTelemetry`, I might consider something like `telemetry.application`, e.g. `telemetry.application.name`, etc.

Reply:
Then we could have `telemetry.performance`, `telemetry.server`, etc

Reply:
done